docker/api/build.py

```diff
@@ -276,10 +276,20 @@ def build(self, path=None, tag=None, quiet=False, fileobj=None,
         return self._stream_helper(response, decode=decode)
 
     @utils.minimum_version('1.31')
-    def prune_builds(self):
+    def prune_builds(self, filters=None, keep_storage=None):
```

Comment:
There's also `all`, which is a boolean and was added at the same time as `filters` and `keep-storage` - let's add that as well

Reply:
Done!